src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/custom.py

```diff
@@ -6,21 +6,143 @@
 import datetime
 import os
 from azure.cli.core.util import get_file_json, CLIError
+from azure.mgmt.monitor.models import ConditionOperator, TimeAggregationOperator
 
 # 1 hour in milliseconds
 DEFAULT_QUERY_TIME_RANGE = 3600000
 
 # ISO format with explicit indication of timezone
 DATE_TIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 
+# region Alerts
+
+operator_map = {
+    '>': ConditionOperator.greater_than.value,
+    '>=': ConditionOperator.greater_than_or_equal.value,
+    '<': ConditionOperator.less_than,
+    '<=': ConditionOperator.less_than_or_equal
+}
+
+
+aggregation_map = {
+    'avg': TimeAggregationOperator.average.value,
+    'min': TimeAggregationOperator.minimum.value,
+    'max': TimeAggregationOperator.maximum.value,
+    'total': TimeAggregationOperator.total.value,
+    'last': TimeAggregationOperator.last.value
+}
+
+
+def _parse_actions(actions):
+    """ Actions come in as a combined list. This method separates the webhook actions into a
+        separate collection and combines any number of email actions into a single email collection
+        and a single value for `email_service_owners`. If any email action contains a True value
+        for `send_to_service_owners` then it is assumed the entire value should be True. """
+    from azure.mgmt.monitor.models import RuleEmailAction, RuleWebhookAction
+    actions = actions or []
+    email_service_owners = None
+    webhooks = [x for x in actions if isinstance(x, RuleWebhookAction)]
+    custom_emails = set()
+    for action in actions:
+        if isinstance(action, RuleEmailAction):
+            if action.send_to_service_owners:
+                email_service_owners = True
+            custom_emails = custom_emails | set(action.custom_emails)
+    return list(custom_emails), webhooks, email_service_owners
+
+
+def _parse_action_removals(actions):
+    """ Separates the combined list of keys to remove into webhooks and emails. """
+    flattened = list(set([x for sublist in actions for x in sublist]))
+    emails = []
+    webhooks = []
+    for item in flattened:
+        if item.startswith('http://') or item.startswith('https://'):
+            webhooks.append(item)
+        else:
+            emails.append(item)
+    return emails, webhooks
+
+
+def create_metric_rule(client, resource_group_name, rule_name, target, condition,
+                       description=None, disabled=False, location=None, tags=None,
```

Comment:
Because if you specify nothing, then it should be enabled (by default). The flag behavior should do the opposite of what "not specifying" does.  To compare:

**enable behavior**
not specified -- enabled
flag specified -- enabled
enabled true -- enabled
enabled false -- disabled

**disable behavior**
not specified -- enabled
flag specified -- disabled
disabled true -- disabled
disabled false -- enabled

I'm fine with either since you can accomplish your scenario either way, but I just don't like that the flag behavior is the same as the default (which is counter to the idea of flags)